server/manager/src/main/java/org/apache/accumulo/master/TabletGroupWatcher.java

```diff
@@ -134,6 +134,37 @@ synchronized boolean isSameTserversAsLastScan(Set<TServerInstance> candidates) {
     return candidates.equals(lastScanServers);
   }
 
+  /**
+   * Collection of data structures used to track Tablet assignments
+   */
+  private static class TabletLists {
```

Comment:
Introducing this class is really nice, really reduces a lot of methods parameters.